.clang-format

```diff
@@ -35,5 +35,6 @@ PenaltyBreakBeforeFirstCallParameter: 1
 PenaltyReturnTypeOnItsOwnLine: 200
 SpaceAfterCStyleCast: 'true'
 SpacesBeforeTrailingComments: '2'
+IndentPPDirectives: AfterHash
```

Comment:
Note that individual project could have a different configuration file since after copying the file they could edit it.

Reply:
The idea is to update them all (and the formatting) after this is merged (they should always have the same contents).

Reply:
I don't think this approach is the best, but this is out of the scope for this PR.

IMO there should exist defaults and each project should be able to override them.

Reply:
This can be reviewed again after the "legacy mode" is thrown away and `clang-format` is ubiquitous

Reply:
We could like that today.
In that scerio the projects opt-in `clang-format` with some comfiguration in the `.isort.cfg` (note that isort can read options from `setup.cfg` and `.editorconfig` also).
Actually I will make `clang-format` the default and projects should opt to use `legacy mode` while porting.
IMO if "black" like behavior is used for how we configure `clang-format` that should also be applied to how we configure `isort`.